README.md

```diff
@@ -117,9 +117,7 @@ the step method returns a RoutingResult object with the following informations d
 ```
 res.time         # the datetime of step  
 res.isochrones   # all points reached at a specified datetime
-res.progress     # the calculation progress
-res.path         # when the end track point is reached contains the route path
-res.position    
+res.progress     # the calculation progress 
```

Comment:
why did you remove path and position?

Reply:
only because before routing.end path and position are void and become significant only when algorithm ends

Reply:
That's not true; if the track has more than two points, path contains middle points even if the complete routing is not done.